rfcs/0173-hotline.md

```diff
@@ -0,0 +1,234 @@
+# RFC nixos hotline
+
+---
+feature: NixOS PR hotline
+start-date: 2024-04-01
+author: trollulus
+co-authors: riotbib, rtrollreal, b-kenji, zmberber, kmein
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Bring the Power of German bureaucracy to Nixpkgs by using a sophisticated telephone system to interact with pull requests.
```

Comment:
It might be worth explaining why this RFC was written in English instead of the more widely used German language 🤔